ic-cdk-timers/Cargo.toml

```diff
@@ -10,23 +10,19 @@ description = "Timers library for the Rust CDK."
 homepage = "https://docs.rs/ic-cdk"
 documentation = "https://docs.rs/ic-cdk-timers"
 readme = "README.md"
-categories = [
-    "api-bindings",
-    "data-structures",
-    "no-std",
-    "development-tools::ffi",
-]
+categories = ["api-bindings", "data-structures", "no-std", "development-tools::ffi"]
 keywords = ["internet-computer", "dfinity", "canister", "cdk"]
 include = ["src", "Cargo.toml", "LICENSE", "README.md"]
 
 [dependencies]
-candid.workspace = true
+futures.workspace = true
```

Comment:
It seems that we no longer depends on `futures`.
Should be safe to remove it here and in the root `Cargo.toml`.

Reply:
It's used in e2e-tests, so not the root.